packages/draw/src/canvas.ts

```diff
@@ -100,36 +108,57 @@ export class Canvas {
     }
   }
 
+  public clear() {
+    this.context?.clearRect(
+      0,
+      0,
+      Number(this.canvasElement.attr('width')),
+      Number(this.canvasElement.attr('height')),
+    );
+  }
+
   public drawFrame({
     zoomer,
     drawables,
-    activeDrawables,
+    config,
   }: {
     zoomer: Zoomer;
     drawables: Drawable[];
-    activeDrawables: Drawable[];
+    config: {
+      highlight: HighlightVariant;
+    };
   }) {
     if (!this.context) return;
 
     this.context.save();
 
-    this.context.clearRect(
-      0,
-      0,
-      Number(this.canvasElement.attr('width')),
-      Number(this.canvasElement.attr('height')),
-    );
     this.context.translate(zoomer.x, zoomer.y);
     this.context.scale(zoomer.k, zoomer.k);
 
-    drawables.forEach((d) => {
-      const highlight = activeDrawables.includes(d)
-        ? 'active'
-        : activeDrawables.length > 0
-        ? 'dimmed'
-        : 'normal';
-      d.draw(this, highlight);
-    });
+    if (config.highlight === 'dimmed') {
+      this.context.globalAlpha = 0.4;
```

Comment:
Should this be externalized somewhere? Maybe via the styles config?

Reply:
Yes for sure!